docs/pages/getting_started/usage.md

```diff
@@ -80,9 +80,10 @@ This can be useful to reduce compile times if the header only build is being use
 
 ## Creating a Format Object
 
-Currently two sparse data formats are supported:
+Currently three sparse data formats are supported:
```

Comment:
So this will now collide with the latest updates to getting_started, I suggest you just accept all from develop.